Add cask for Fira Code v0.3

martinbmadsen opened the pull request:
The Firefox Fira monospaced font with programming ligatures, based on Hasklig.

vitorgalvao commented on font-fira-code.rb:
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'font-fira-code' do
+  version '0.3'
+  sha256 :no_check
```
`sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

vitorgalvao commented on font-fira-code.rb:
```diff
@@ -0,0 +1,11 @@
+  license :ofl
+
+  font 'FiraCode-Regular.otf'
+
```
Please remove this empty line.

martinbmadsen commented:
Updated to have sha256... my bad. I was using https://github.com/caskroom/homebrew-fonts/blob/master/CONTRIBUTING.md as an example, and noticed that the font-inconsolata didn't have a sha256 check because it was not a container, so I thought the same was true for this font.

But anyway, updated now!

vitorgalvao commented:
> noticed that the font-inconsolata didn't have a sha256 check because it was not a container

It has nothing to do with that. It doesn’t have a checksum because it doesn’t have a version, which means it is updated in place, and the cask would then break constantly.
